_pytest/capture.py

```diff
@@ -308,6 +308,14 @@ class FDCapture:
     """ Capture IO to/from a given os-level filedescriptor. """
 
     def __init__(self, targetfd, tmpfile=None):
+        # ensure readline is imported so that it attaches to the correct
+        # stdio handles on Windows
+        if targetfd in (0, 1, 2):
```

Comment:
I don't know that I'll have time anytime soon to finish working on the actual fixes to readline, so there may be nothing to link to for a while, but I'll add a link back to this PR.